src/scripts/main.js

```diff
@@ -1,7 +1,24 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector('body');
+  const notification = document.createElement('div');
+  const h2 = document.createElement('h2');
+  const p = document.createElement('p');
+
+  h2.classList.add('title');
+
+  notification.classList.add('notification');
+  notification.classList.add(type);
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  h2.textContent = title;
+  p.textContent = description;
+  notification.append(h2, p);
+
+  body.appendChild(notification);
+  setTimeout(() => (notification.style.display = 'none'), 2000);
```

Comment:
The current logic hides the notification after 2 seconds by setting its display to 'none'. Consider removing the notification from the DOM entirely using `body.removeChild(notification)` after the timeout to free up memory and avoid potential clutter in the DOM.